contrib/scripts/generate_debian_changelog.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
I'm strongly against auto-generating changelog entries from git commits.  the point of a change log is to convey important changes to the user.  The point of git commit log entries is to explain a particular change in isolation.  This just makes everything a mess.

Reply:
This is a packaging changelog, separate from any sort of user-facing upstream changelog.

See the [upstream debian libfabric](https://salsa.debian.org/hpc-team/libfabric/-/blob/debian/master/debian/changelog?ref_type=heads) changelog for an example.

Reply:
yes, same objection.  The very thing you link to doesn't appear to be autogenerated?

Reply:
What do you propose we do here instead?

Reply:
Why is it a problem to update by hand?  That appears to be what Debian is doing?

Reply:
Because the version of the package is derived from the version stated in the topmost changelog entry, and we want to be capable of testing that package generation works on any arbitrary commit.